homeworks/sasha.demidenko_SashaDemi/GamesProject/games.html

```diff
@@ -14,9 +14,49 @@
                     <li>
                         <a href="index.html" class="games_header--li">About</a>
                     </li>
+                    <li>
+                        <a href="login.html" class="games_header--li">Login</a>
+                    </li>
                 </ul>
             </nav>
         </header>
+        <section>
```

Comment:
As the form is a flex container, better to add `margin-left: auto` to its direct child, label, than to fix this margin for forms grandchild
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/4159a5ce-5305-4cce-84a3-39f12db12948)

Reply:
And this margin slaps out of the header and shifts it out of the page edge.
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/0843d5c2-55c5-409f-8366-29541246502d)

Reply:
Have you noticed, that all these containers have the same side indentation? So, proly you can add equal paddings to them 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/c9acc734-ef6e-4e57-9611-99176788ee7e)

Reply:
Thanks for your corrections. I tried to correct your comments, but perhaps I missed something because I don’t know how to solve it yet, but I’m learning, please be understanding, thank you

Reply:
So, please don't get my comment as an offense. I just like to point your attention to it and give some advice. Our common aim here is to teach you some useful stuff.🕊️

Reply:
Ok, so good, so far 👍